dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -661,6 +661,7 @@
 			interrupt-names = "COMMON";
 			status = "disabled";
 			label = "ETH_0";
+			phy-addr = <2>;
```

Comment:
Shouldn't this be in the previous commit? Looks like a rebase went awry somehow.

Reply:
I placed it here because this commit is where I actually enabled the enet driver for the RT11xx, before this commit the driver will fail to build for the 11xx SOC series. I'll go ahead and move it to the previous commit.